contracts/src/v0.8/upkeeps/CronUpkeep.sol

```diff
@@ -92,11 +90,8 @@ contract CronUpkeep is KeeperCompatibleInterface, KeeperBase, ConfirmedOwner, Pa
     );
     validate(id, tickTime, target, handler);
     s_lastRuns[id] = block.timestamp;
-    (bool success, bytes memory payload) = target.call(handler);
-    if (!success) {
-      revert CallFailed(id, getRevertMsg(payload));
-    }
-    emit CronJobExecuted(id, block.timestamp);
+    (bool success, ) = target.call(handler);
+    emit CronJobExecuted(id, success);
```

Comment:
we _could_ include revert data in the event, but I'm unsure of the value add

Reply:
@RyanRHall you added a revert reason on registry right?

Reply:
I would say to not have it unless we see a need. It adds extra gas and code size

Reply:
also our previous method `getRevertMsg` only works `require` errors and not really for custom error signatures